cfgov/cfgov/settings/base.py

```diff
@@ -700,7 +700,10 @@
     'HMDA_OUTAGE': [
         {'condition': 'parameter', 'value': 'hmda-outage', 'required': True},
         {'condition': 'path matches', 'value': r'^/data-research', 'required': True}
-    ]
+    ],
+    
+    #Add HowTo schema markup to answer page
```

Comment:
Minor: Could there be a space added between # and the text?

Reply:
Updated in 6748d40